package.json

```diff
@@ -5,14 +5,56 @@
   "main": "lib/shopify.js",
   "repository": "git@github.com:Shopify/js-buy-sdk.git",
   "scripts": {
-    "build": "node ./scripts/build",
-    "prepublish": "node ./scripts/prepublish",
+    "start": "npm run dir-setup-dist-test && npm run dir-setup-dist && concurrently 'npm run watch-test' 'npm run build-test-sdk' 'http-server -p 4200 .dist-test/'",
+    "test": "npm run dir-setup-dist-test && npm run dir-setup-dist && npm run build-test && npm run build-test-sdk && testem ci",
+    "test-examples": "npm run dir-setup-dist-test && npm run dir-setup-dist && npm run build-test && npm run build-test-sdk && npm run cp-examples && npm run css-examples && npm run test-run-example",
+    "test-ci": "npm run dir-setup-dist-test && npm run dir-setup-dist && npm run build-test && npm run build-test-sdk && testem ci -R xunit > tests.xml",
+    "examples": "npm run dir-setup-dist-test && npm run dir-setup-dist && npm run cp-examples && npm run css-examples && http-server -p 4200 .dist-test/",
+    "build": "npm run dir-setup-dist && npm run build-all && npm run minify-all",
```

Comment:
`test`, `test-examples` and `test-ci` all seem to start the same way, would it make sense to pull that out?
eg:
```json
{
  "...": null,
  "test-setup":    "npm run dir-setup-dist-test && npm run dir-setup-dist && npm run build-test && npm run build-test-sdk",
  "test":          "npm run test-setup && testem ci",
  "test-examples": "npm run test-setup && npm run cp-examples && npm run css-examples && npm run test-run-example",
  "test-ci":       "npm run test-setup && testem ci -R xunit > tests.xml",
  "examples":      "npm run dir-setup-dist-test && npm run dir-setup-dist && npm run cp-examples && npm run css-examples && http-server -p 4200 .dist-test/",
  "build":         "npm run dir-setup-dist && npm run build-all && npm run minify-all",
}
```